app/models/service_template.rb

```diff
@@ -10,6 +10,20 @@ class ServiceTemplate < ApplicationRecord
     "storage"         => _("Storage")
   }.freeze
 
+
+  CATALOG_ITEM_TYPES = {
```

Comment:
@h-kataria Is the idea to change the [catalog_item_types](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/catalog_controller.rb#L130) method from the classic UI repo and have it reference this constant?  We would not want to have this defined in multiple places.

@Fryguy @dclarizio Any thoughts here?

Ultimately this list needs to make it's way into the pluggable providers.  @durandom